client/lib/plans/features-list.js

```diff
@@ -102,10 +103,15 @@ export const FEATURES_LIST = {
 	[ constants.FEATURE_ALL_PREMIUM_FEATURES ]: {
 		getSlug: () => constants.FEATURE_ALL_PREMIUM_FEATURES,
 		getTitle: () => i18n.translate( 'All Premium features' ),
-		getDescription: () =>
-			i18n.translate(
-				'Including unlimited premium themes, advanced design and monetization options, simple payment buttons, and a custom domain name for one year.'
-			),
+		getDescription: () => {
+			isEnabled( 'earn/pay-with-paypal' ) ?
+				i18n.translate(
+					'Including unlimited premium themes, advanced design and monetization options, Pay with PayPal buttons, and a custom domain name for one year.'
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.